### DISCUSSION TITLE
Typo

### DISCUSSION THREAD

**Discussion by netmarkjp** [topotal/waroom-roadmap#10] [Bug Report]
![image](https://user-images.githubusercontent.com/87943/150044426-f7bb8fc9-27fc-4adc-b930-19fa0635f1e7.png)

てにをはですが `インシデントレスポンスを` かなと思います。

**Comment by sawa-zen** [comment]
@netmarkjp これはお恥ずかしい 🙇🏻 ありがとうございます 🙏🏻

**Comment by sawa-zen** [comment]
修正して反映いたしました! ご指摘ありがとうございました 👍🏻
